sys/Makefile.dep

```diff
@@ -648,9 +648,7 @@ ifneq (,$(filter gcoap,$(USEMODULE)))
   USEMODULE += ztimer_usec
   USEMODULE += event_callback
   USEMODULE += event_timeout_ztimer
-  ifneq (,$(filter openwsn%,$(USEMODULE)))
-    USEMODULE += openwsn_sock_udp
-  endif
```

Comment:
This removal seems unrelated.

Reply:
Its in the commit message [34eb656](https://github.com/RIOT-OS/RIOT/pull/17944/commits/34eb65642ccb53f58e0ffe5a8edabd34d4c30696)